### PR TITLE
feat: add tooltips in fixed deposit products

### PR DESCRIPTION
**Pull request by PC-11-00**
## Description
Added ToolTips in Fixed Deposit Products that provided a valuable aid to understanding functionality of the system .

## Related issues and discussion
#1654 

## Screenshots, if any

https://user-images.githubusercontent.com/76156941/216444727-a2f262f5-ea48-472f-a63b-d882357f1875.mov

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by rish106** [comment]
Can you squash the commits into a single one and follow the [commit message conventions](https://github.com/openMF/web-app/blob/master/.github/COMMIT_MESSAGE.md)?

**Comment by PC-11-00** [comment]
@rish106 @spider0061 have a look

File: src/app/products/fixed-deposit-products/fixed-deposit-product-stepper/fixed-deposit-product-interest-rate-chart-step/fixed-deposit-product-interest-rate-chart-step.component.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { Component, OnInit, Input } from '@angular/core';
 import { FormGroup, FormBuilder, Validators, FormArray } from '@angular/forms';
 import { MatDialog } from '@angular/material/dialog';
 import { animate, state, style, transition, trigger } from '@angular/animations';
+import {TooltipPosition} from '@angular/material/tooltip';
```
Comment (rish106) [inline src/app/products/fixed-deposit-products/fixed-deposit-product-stepper/fixed-deposit-product-interest-rate-chart-step/fixed-deposit-product-interest-rate-chart-step.component.ts:6]:
```suggestion
import { TooltipPosition } from '@angular/material/tooltip';
```

File: src/app/products/fixed-deposit-products/fixed-deposit-product-stepper/fixed-deposit-product-settings-step/fixed-deposit-product-settings-step.component.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { Component, OnInit, Input } from '@angular/core';
 import { FormGroup, FormBuilder, FormControl, Validators } from '@angular/forms';
+import {TooltipPosition} from '@angular/material/tooltip';
```
Comment (rish106) [inline src/app/products/fixed-deposit-products/fixed-deposit-product-stepper/fixed-deposit-product-settings-step/fixed-deposit-product-settings-step.component.ts:3]:
```suggestion
import { TooltipPosition } from '@angular/material/tooltip';
```

File: src/app/products/fixed-deposit-products/fixed-deposit-product-stepper/fixed-deposit-product-terms-step/fixed-deposit-product-terms-step.component.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import { Component, OnInit, Input } from '@angular/core';
 import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import {TooltipPosition} from '@angular/material/tooltip';
```
Comment (rish106) [inline src/app/products/fixed-deposit-products/fixed-deposit-product-stepper/fixed-deposit-product-terms-step/fixed-deposit-product-terms-step.component.ts:3]:
```suggestion
import { TooltipPosition } from '@angular/material/tooltip';
```

**Review by rish106 — CHANGES_REQUESTED** [review]
Kindly make these changes and also change the title of this pull request to the same as commit message

**Comment by PC-11-00** [comment]
@rish106 have a look, I changed the name of the PR which matches with the commit, and also made the suggested changes

**Comment by spider0061** [comment]
LGTM!

**Pull request merged**